design-patterns/domain-driven-design/pattern2/README.md

```diff
@@ -0,0 +1,205 @@
+# Pattern 2: Single Domain, Multiple Bounded Contexts, and a Single Uni
+
+**Prerequisite:** Please familiarize yourself with the Pattern 1 before you practice the Pattern 2.
```

Comment:
```suggestion
**Prerequisite:** Please familiarize yourself with the Pattern 1 before you move to Pattern 2.
```

Reply:
The term `practice` isn't resonating with me.  There's nothing to practice here...just examples to review/learn from/use as a guide.

Reply:
I will change it to review. But, a person has to do hands-on by creating Uni, running queries, and validating results.